sdk/analytics/build.gradle

```diff
@@ -14,8 +14,8 @@ afterEvaluate {
         publications {
             // Creates a Maven publication called "release".
             release(MavenPublication) {
-                from components.release
-                groupId = ext.groupId
+                from components[ext.publishBuildVariant]
```

Comment:
Might want to publish a debug variant too, but at least you can publish to local maven and use that from the test app. The _sample_ app will be able to use any build variant because depends on the projects instead of artifacts